### PR TITLE
WIP #248 Gradle files refactoring

### PR DESCRIPTION
**Pull request by frankscholten**

### CODE REVIEW

**Comment by mwl** [comment]
Too few Maven files in my opinion

**Comment by adam-sandor** [comment]
LGTM

File: build.gradle [hunk 1]
```diff
@@ -26,9 +25,48 @@ plugins {
...
+subprojects {
+
+    group = "com.containersol.minimesos"
+    version = rootProject.version.toString()
+
+    repositories {
+        mavenCentral()
+        mavenLocal()
+        maven {
+            url "https://jitpack.io"
```
Comment (mwl) [inline build.gradle]: I would be a little bit careful with putting jitpack at the very top level. _Potentially_ someone from the outside can screw up everything if they just manage to amend a single dependency. Which isn't too far fetched IMO.

Comment (mwl) [inline build.gradle]: But anyway. Nothing has been made worse than it already is by this change. Still it's something to keep an eye out for.

**Pull request merged**